### ISSUE TITLE
Why pnpm instead of npm or yarn?

### ISSUE THREAD

**Issue by Soleone** [pengx17/logseq-plugin-template-react#667]
Is `pnpm` really necessary instead of being able to use `npm` or `yarn`? If so could it be clarify e.g. in the Readme why?

**Comment by pengx17** [comment]
No, using `pnpm` is not necessary; it is simply my personal preference. I believe you can remove the lock file after cloning the repository and then use your preferred package manager.

Pull requests are welcome!
